### ISSUE TITLE
Write out any screen-reader content differences

### ISSUE THREAD

**Issue by zachberry** [ucfopen/Obojobo-Docs#18]
For example, if a MathEquation reads labels differently then we should have a note for MathEquation for what the screen reader will say.
